addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1899,17 +1899,19 @@ export class OdooEditor extends EventTarget {
      * @returns {boolean} true if a table was deselected
      */
     deselectTable() {
+        const tds = this.editable.querySelectorAll('.o_selected_table, .o_selected_td');
+        if (!tds.length) {
+            return;
```

Comment:
Note that this is a spec change. The function used to return `boolean`, now it returns `true | undefined`. If you don't mind, I'd rather keep the return type consistent, especially in stable. 😉 

There are a few typos in this commit message, notably "top ," (extra space) and "don't get caled" (missing a L).